department-of-veterans-affairs/caseflow
# confirmation of response letters

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-29426)

# Description
Display of response letters section on the step 3 oof confirmation page.

## Acceptance Criteria

## Testing Plan
1. Login as a Mail team user MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER (VACO)
2. From the App selector, select and click on Queue
3. In Your cases page, click on Switch views and select Correspondence Cases
4. Now you have to see the Correspondence Cases page.
5. Click in one case, you can see the Review Package page
6. Scroll to the bottom of the page and click on Create record button.
7. Now you can see the first tab on intake page and see the + Add Letter button.
8. When you click on + Add Letter a new letter appears, and the continue button disappears until you complete the entire 
form.
9. You can Add only 3 letters for correspondence or less when you have 3 letters the + Add Letter button is not clickable anymore.
10. Click Continue button and then you will navigate to the step 2 page.
11. Click continue on the step 2 page will be navigated to step 3 of the confirmation page
12. Response letters section will be displayed with the add letter selections that has been made on step 1.

<img width="1661" alt="Screen Shot 2024-02-08 at 1 48 19 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/128258952/abcb74d3-ec3e-4593-8f06-a6dfea8f33cd">




# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
